src/main/resources/main-spring-config.xml

```diff
@@ -70,18 +79,21 @@
 				<ref bean="stopHandler" />
 				<ref bean="fallbackHandler" />
 				<ref bean="helpHandler" />
-				<ref bean="welcomeSymtomsCheckerHandler" />
+				<ref bean="bodyLocationHandler" />
 				<ref bean="launchRequestHandler" />
+				<ref bean="yesIntentHandler" />
+				<ref bean="noIntentHandler" />
 				<ref bean="sessionEndedHandler" />
 			</list>
 		</property>
 	</bean>
 
-		<bean id="diagnosisClient"
+	<bean id="diagnosisClient"
 		class="com.symtoms.checker.alexa.integration.client.impl.DefaultDiagnosisClient" />
-	
-	<!-- Importing separate Spring Rest Template configurations -->
-	<!-- Importing separate Spring Integration configurations -->
-	<import resource="classpath:resources/integration/symtoms-checker-common-integration.xml"/>	
-	
+	<bean id="symtomsCheckerService"
+		class="com.symtoms.checker.alexa.service.impl.DefaultSymtomsCheckerService" />
+			
+	<bean id="symtomsRestTemplate" class="org.springframework.web.client.RestTemplate" />
```

Comment:
Do we need this Rest Template now !! I think currently we are using SDK Client provided by API Medic.

Reply:
Not for this Release, we might require in next release, if we integrate with practo api